.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
```

Comment:
Can we have a comment at the top of this file with what it's doing so it's easy to identity?

Reply:
Done!